### PR TITLE
Add new admin section

### PR DESCRIPTION
**Pull request by ZMYaro**

### CODE REVIEW

**Comment by derekw023** [comment]
Functionally seems good. The home page throws an error 500 if there is no home page content stored, but I don't think there's much point in adding in initialization code, that condition should not happen after the first time we put content in there. I approve as is.

**Review by dexw25 — APPROVED** [review]
Functionally seems good. The home page throws an error 500 if there is no home page content stored, but I don't think there's much point in adding in initialization code, that condition should not happen after the first time we put content in there. I approve as is.

File: main.py [hunk 1]
```diff
@@ -21,6 +21,10 @@ def get(self):
 		template_vals = generate_base_template_vals(self)
 		template_vals['page'] = 'home'
 		
+		# Get the home page text.
+		home_content = PageContent.query(PageContent.page == 'home').get()
+		template_vals['home_html'] = home_content.html_text
```
Comment (ZMYaro) [inline main.py]: @derekw023: I think the bug you found should go away if this line is changed to
```template_vals['home_html'] = '' if not home_content else home_content.html_text```
but I will wait and test it after work before committing it.

**Comment by ZMYaro** [comment]
@derekw023: Fixed!

**Comment by dexw25** [comment]
Fix is good, still good to merge.

**Pull request merged**